.github/workflows/publish_docker.yml

```diff
@@ -0,0 +1,41 @@
+name: publish docker
+
+on:
+  push:
+    paths:
+      - "src/**"
+      - ".github/workflows/publish_docker.yml"
+    branches:
+      - main
+  pull_request:
+    paths:
+      - "src/**"
+      - ".github/workflows/publish_docker.yml"
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        include:
+          - tag: "7.0" #Debian 11
+            file: Dockerfile
+          - tag: 7.0-jammy # Ubuntu 22.04
+            file: Dockerfile
```

Comment:
Do we need two different builds for them? If they use the same glibc I'd assume one build would be sufficient?
When we release bmx, ideally we release as few binaries as possible to support as many OSes as possible.

Reply:
I didn't realize that `glibc` package was what mattered and assumed they would need to be separate for each OS. I can spin up an ec2 instance for each and try them